### PR TITLE
Fix a problem that could not update without changing email

### PR DESCRIPTION
**Pull request by suzuki**
## About

- I injected a problem by #257.
- It is impossible update self information without changing email.

## How to fix the problem

- Added self `user._id` into query to find same email address

### CODE REVIEW

File: lib/routes/me.js [hunk 1]
```diff
@@ -89,7 +89,8 @@ module.exports = function(crowi, app) {
         return res.render('me/index', {});
       }
 
-      User.findOne({email: email}, (err, existingUserData) => {
+      User.findOne({email: email, _id: {'$ne': userData._id}}, (err, existingUserData) => {
```
Comment (sotarok) [inline lib/routes/me.js]: hmm ... ?
if the user couldn't find here, then the process is not into this if statement, 
it means ... lang and name are also not updated?

Comment (sotarok) [reply lib/routes/me.js]: it should be ...

- user changes their email address to another one, and it's not duplicated => email can be changed
- user changes email address and it's duplicated => email can't be updated
- anyway, others (name and lang) can be updated

**Comment by sotarok** [comment]
@suzuki it has also bug. can you review it?

File: lib/routes/me.js (hunk 1 above)
Comment (abouthiroppy) [reply lib/routes/me.js]: First of all, we should write exception handling, right?😉

Comment (suzuki) [reply lib/routes/me.js]: when email was invalid, `lang` and `name` are not updated.
https://github.com/crowi/crowi/pull/259/files#diff-1972c53f45200d68ac6d5e645409aca1L87

here is same logic.

Comment (sotarok) [reply lib/routes/me.js]: I misunderstood about this logic.
It would be ok.

**Pull request merged**